frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/MainNavbarView.tsx

```diff
@@ -189,29 +164,29 @@ export function MainNavbarView({
             >
               {t`Home`}
             </PaddedSidebarLink>
-            {showOnboardingLink && (
-              <PaddedSidebarLinkDismissible
-                icon="learn"
-                right={
-                  <Tooltip label={t`Hide page`} offset={16} position="right">
-                    <Icon
-                      className="dismiss"
-                      name="eye_crossed_out"
-                      onClick={dismissOnboardingLink}
-                    />
-                  </Tooltip>
-                }
-                url={ONBOARDING_URL}
-                isSelected={isOnboardingPageSelected}
-                onClick={() => trackOnboardingChecklistOpened()}
-              >
-                {/* eslint-disable-next-line no-literal-metabase-strings -- We only show this to non-whitelabelled instances */}
-                {t`How to use Metabase`}
-              </PaddedSidebarLinkDismissible>
-            )}
+
             {showUploadMenu && <DwhUploadMenu />}
           </SidebarSection>
 
+          {isNewInstance && (
+            <SidebarSection>
+              <ErrorBoundary>
+                <GettingStartedSection nonEntityItem={nonEntityItem}>
+                  {examplesCollection && (
+                    <Tree
+                      data={[examplesCollection]}
+                      selectedId={collectionItem?.id}
+                      onSelect={onItemSelect}
+                      TreeNode={SidebarCollectionLink}
+                      role="tree"
+                      aria-label="examples-collection-tree"
```

Comment:
Decided to keep the tree inside of this component to:
1. avoid prop drilling
2. make testing the `GettingStartedSection` component easier